reproducibility-with-guix.org

```diff
@@ -356,13 +362,28 @@ That looks much better. As you can see from the code, a list of inputs is a bit
 
 (format #t "Package: ~a\n"
         (package->specification gcc-toolchain))
-(format #t "Inputs: ~a\n"
+(format #t "Inputs : ~a\n"
         (unique-inputs (package-direct-inputs gcc-toolchain)))
+(format #t "Internals: ~a\n"
```

Comment:
Do you think this is useful information? I see these labels are implementation details of the build script.

Reply:
When reading, I checked the outputs and the dependencies.

My point is: `gcc` as 3 outputs too but appears once because only the `out` outputs is in the dependency list.

Otherwise, I agree these labels are not reliable and can considered as an historical mistake. ;-) 
The information is only for pedagogical purpose. It helped me but I do know if it is useful.